### DISCUSSION TITLE
Osu!lazer Android Landscape mode

### DISCUSSION THREAD

**Discussion by Arin4969** [ppy/osu#15088] [Ideas] [answered]
why does my osu!lazer Android is portrait mode and has a super tiny fonts. I also try "auto rotate" and it works changing to landscape mode but the fonts was the problem cause it was tiny and hope the the dev will change the default screen mode to landscape not portrait cause it might be bothersome if I accidentally mode my phone while playing.

**Comment by bdach** [comment, marked as answer]
> also try "auto rotate" and it works changing to landscape mode

disabling portrait rotation lock is the recommended solution for now.

> but the fonts was the problem cause it was tiny

you can change UI scaling in settings.

> the dev will change the default screen mode to landscape not portrait

landscape is the only allowable rotation mode during gameplay already.

the mobile experience is known to be subpar for now but the current focus is the desktop client. mobile improvements will come later.

**Reply by Arin4969** [reply]
Oh thank but where I can find the disabling portrait lock? Is it in game settings?

**Reply by bdach** [reply]
i meant the system-wide rotation lock settings, most frequently accessible via quick settings in the android status bar.
